src/plugins/intel_gpu/include/intel_gpu/runtime/layout.hpp

```diff
@@ -126,58 +126,107 @@ inline data_types element_type_to_data_type(ov::element::Type t) {
     }
 }
 
+constexpr size_t SHAPE_RANK_MAX = 9;
+
 /// @brief Represents data padding information.
 struct padding {
     /// @brief Filling value for padding area.
     float filling_value() const { return _filling_value; }
 
     /// @brief Gets lower padding sizes. For spatials, it means size of left (X) and top (Y) padding.
     /// @return Tensor with padding for top/left/lower bounds of data.
-    tensor lower_size() const { return _lower_size; }
+    std::vector<tensor::value_type> lower_size(const size_t max_size = 0) const {  // TODO: get shape size
```

Comment:
Why do still do a copy here? In think we can just make `_lower_size` array public and access elements directly

Reply:
um, sounds okay.

Reply:
done